packages/react-heartwood-components/src/components/List/components/SortableList/SortableList.tsx

```diff
@@ -1,47 +1,61 @@
-// @flow
 import React, { Component, Fragment } from 'react'
 import cx from 'classnames'
 import {
 	SortableContainer,
 	SortableElement,
 	arrayMove
 } from 'react-sortable-hoc'
+import { IListProps } from '../../List'
 import ListHeader from '../ListHeader/ListHeader'
-import ListItem from '../ListItem/ListItem'
-import type { Props as ListItemProps } from '../ListItem/ListItem'
-import type { Props as ListProps } from '../../List'
+import ListItem, { IListItemProps } from '../ListItem/ListItem'
 
-type Props = {
+export interface ISortableListProps extends IListProps {
 	/** onConfirm callback */
-	onConfirm?: Function,
-	...ListProps
+	onConfirm?: Function
+
+	/** Optional classname for the parent */
+	parentClass?: string
+
+	/** Optional; disables sorting */
+	disabled?: boolean
+
+	/** Callback when sorting starts */
+	onSortStart: Function
```

Comment:
For all `Function` properties define the shape of the expected input. `(a: string) => void` etc. Make a new interface if it's sufficiently complex / reusable